NOTICE.txt

```diff
@@ -532,6 +532,30 @@ AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
 LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 SOFTWARE.
+--------------------------------------------------------------------
+Dependency: github.com/elastic/ecs
+Revision: 69de90eb6493e0804405321f48adfdfa488d6498
```

Comment:
My first though was to fix it to Beta2 but at this stage the go code was not in yet :-(

Reply:
Luckily the code generated from this commit is exactly the same as what would be in beta2 if I backported (I started a backport and realize they’d be the same).d